models/us_nature_gerda/code/download.py

```
@@ -0,0 +1,93 @@
+    "european_muni_unharm": "european_elections/final/european_muni_unharm",
+    "european_muni_harm": "european_elections/final/european_muni_harm",
+    # --- Crosswalks (source for br_bd_diretorios_de geography) ---
+    "ags_crosswalks": "crosswalks/final/ags_crosswalks",
+    "cty_crosswalks": "crosswalks/final/cty_crosswalks",
+}
+
+
+def download(name, path):
+    url = f"{BASE}/{path}.csv?download="
+    dest = os.path.join(INPUT, f"{name}.csv")
+    req = urllib.request.Request(url, headers={"User-Agent": "curl/8"})
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)